### PR TITLE
SelfAcessorFixer - class property fix

### PR DESCRIPTION
**Pull request by mnabialek**
It fixes the issue described in https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2831

Hope it won't break anything :)

### CODE REVIEW

File: src/Fixer/ClassNotation/SelfAccessorFixer.php [hunk 1]
```diff
@@ -120,6 +120,11 @@ private function replaceNameOccurrences(Tokens $tokens, $name, $startIndex, $end
                 continue;
             }
 
+            // skip tokens that are part of class property
+            if ($prevToken->isGivenKind(T_OBJECT_OPERATOR)) {
```
Comment (SpacePossum) [inline src/Fixer/ClassNotation/SelfAccessorFixer.php]: you can merge this check into the one above;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.4/src/Fixer/ClassNotation/SelfAccessorFixer.php#L119

```php
            // skip tokens that are part of a fully qualified name
            if ($prevToken->isGivenKind(T_NS_SEPARATOR) || $nextToken->isGivenKind(T_NS_SEPARATOR)) {
                continue;
            }
```

=>

```
            // skip tokens that are part of a fully qualified name or used in class property access
            if ($prevToken->isGivenKind([T_NS_SEPARATOR, T_OBJECT_OPERATOR]) || $nextToken->isGivenKind(T_NS_SEPARATOR)) {
                continue;
            }
```

Comment (mnabialek) [inline src/Fixer/ClassNotation/SelfAccessorFixer.php]: It's done now.

**Comment by SpacePossum** [comment]
please make the `if` statement one line and this is a solid bug fix ready to be merged :)

**Comment by mnabialek** [comment]
@SpacePossum  Done, builds in progress. You really like to use so long lines, don't you?

**Comment by SpacePossum** [comment]
haha well the alternative would indeed be multi line but we have CS rules on those as well. 
Thanks for updating, all good :+1:

**Comment by keradus** [comment]
Thank you @mnabialek.

**Pull request closed without merging**